lib/lpc55-rot-startup/src/lib.rs

```diff
@@ -201,6 +184,30 @@ pub fn startup(
         panic!();
     };
 
+    #[cfg(any(feature = "dice-mfg", feature = "dice-self"))]
```

Comment:
I might make a separate change to drop this cfg check as all our RoT targets now enable one of `dice-mfg` or `dice-self`